<?xml version="1.0"?>
<launch>

    <node pkg="wamv_driver" type="motor_command_interpretter.py" name="motor_command_interpretter_node" >
      <remap from='topic_to_port_motor'          to='LmotorSpeed' />
      <remap from='topic_to_strbrd_motor'        to='RmotorSpeed' />
      <remap from='topic_to_port_bow_thruster'   to='LmotorSpeed_lateral' />
      <remap from='topic_to_strbrd_bow_thruster' to='RmotorSpeed_lateral' />
      <remap from='motor_command'                to='motor_command' />
      <remap from='system_mode'                  to='system_mode' />
    </node>

</launch>
